stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -946,4 +950,4 @@ end
 
 function Base.muladd(A::Diagonal, B::Diagonal, z::Diagonal)
     Diagonal(A.diag .* B.diag .+ z.diag)
-end
+end
```

Comment:
```suggestion
end

```
No need to delete trailing newline (all source files have exactly one trailing newline by convention, enforced by the whitespace CI check)